swupd/create_manifests.go

```diff
@@ -176,10 +176,6 @@ func CreateManifests(version uint32, minVersion bool, format uint, statedir stri
 	var err error
 	var c config
 	c, err = getConfig(statedir)
```

Comment:
Should still return error?

Reply:
I know there is a fallback, but an server.ini we can't read might indicate a problem, so worthwhile failing?

Reply:
Maybe this should be a real failure then. If a server.ini exists but we can't read the file the user had configuration that we don't know about.

I think we should return the error.

Reply:
If this could output a bad manifest or update, we should set the error and return it at the end anyway. If not - then throw it out and ignore it. Optionally one could add a nonzero return value to check and log the error in the calling function.

Reply:
Looks like we all fell all over ourselves to agree we should return the error.

Reply:
+1 :) I will update this PR with more changes then, wanted to get the ball rolling and your opinions on it.